### PR TITLE
[IOPID_2184]

### PR DESCRIPTION
**Pull request by mariateresaventura**
Updated and modified the FAQ in the settings section, both in Italian and English

### CODE REVIEW

File: contextualhelp/data.json [hunk 1]
```diff
@@ -1661,59 +1734,65 @@
...
-            "body": "Go to the 'Profile' section of the IO app. Biometric recognition will already be turned off if your device does not support it, or if it is disabled in the system preferences."
+            "body": "If your phone supports biometric recognition, enable it from system preferences. Once done, also enable it from the 'Settings' of the IO app."
           }
         ]
       },
       {
         "route_name": "PROFILE_PRIVACY_MAIN",
         "title": "Your privacy settings",
-        "content": "Here you can find information about the processing of your personal data and IO's terms of use. More specifically, you can read the Privacy Policy, download a copy of the data associated with your account or delete it and give or withdraw permission to the app to collect your usage data.",
+        "content": "Here you can find all the information regarding the processing of your personal data. You can also request a copy of the data associated with your profile or request that it be deleted.",
```
Comment (shadowsheep1) [inline contextualhelp/data.json:1787]: The comma (`,`) at the end is wrong. Maybe it's shifted from somewhere else during editing?

File: contextualhelp/data.json [hunk 2]
```diff
@@ -423,17 +454,27 @@
...
           }
         ]
+      },      
+      {
+        "route_name": "PIN_SCREEN",
+        "title": "Codice di sblocco",
+        "content": "**Il codice di sblocco è il codice di sicurezza che mantiene al sicuro il tuo accesso all’app.**.\nSe è la prima volta che entri nell'app, imposta liberamente il tuo codice numerico e memorizzalo in modo da poterlo utilizzare durante tutti i successivi accessi. Se l'hai già impostato e non lo ricordi, puoi selezionare 'Hai dimenticato il codice di sblocco?'e impostare un nuovo codice.\nIl codice di sblocco è legato al dispositivo su cui l'hai impostato: se lo cambi, dovrai impostarlo nuovamente al primo accesso all'app IO."
+      },
+      {
+        "route_name": "ONBOARDING_TOS",
```
Comment (shadowsheep1) [inline contextualhelp/data.json:475]: This route's repeated twice per language. That's wrong. We should have only one route per language.

**Review by shadowsheep1 — CHANGES_REQUESTED** [review]
@mariateresaventura  I’ve checked the issue and noticed that on line `1810`, the comma (`,`) is placed at the end of the string instead of after the closing curly bracket. Additionally, the `ONBOARDING_TOS` section is repeated twice in both languages.

File: contextualhelp/data.json (hunk 1 above)
Comment (mariateresaventura) [reply contextualhelp/data.json:1787]: Typing error, it was a dot

**Review by shadowsheep1 — APPROVED** [review]
LGTM

**Pull request merged**